packages/react-reconciler/src/__tests__/SchedulingProfilerLabels-test.internal.js

```diff
@@ -140,9 +140,7 @@ describe('SchedulingProfiler labels', () => {
       targetRef.current.click();
     });
     expect(clearedMarks).toContain(
-      `--schedule-state-update-${formatLanes(
-        ReactFiberLane.InputDiscreteLane,
-      )}-App`,
+      `--schedule-state-update-${formatLanes(ReactFiberLane.SyncLane)}-App`,
```

Comment:
Heads up @bvaughn, this change will change the lane label for discrete updates to Sync (because that's what they are now).

Reply:
That's fine, so long as `getLabelsForLanes` is still valid which it looks like it is 👍🏼  Thanks for the heads up though!